src/screens/Login/Login.tsx

```diff
@@ -85,9 +85,7 @@ export const Login = () => {
                     onSubmit={handleSubmit((data) =>
                         loginFunc
                             .mutateAsync(data)
-                            .then((result) =>
-                                navigate(`${ROUTES.PROFILE}/${result.username}`)
-                            )
+                            .then(() => navigate(`${ROUTES.FEED}`))
```

Comment:
```
navigate(ROUTES.FEED)
```

This does not create a new string in memory. It is more performant.

Reply:
Done